--- linux-2.6.18.6/arch/um/os-Linux/sys-i386/registers.c.orig	2007-04-16 12:04:36.930356683 +0200
+++ linux-2.6.18.6/arch/um/os-Linux/sys-i386/registers.c	2007-04-16 12:05:18.533250088 +0200
@@ -13,6 +13,12 @@
 #include "registers.h"
 #include "user.h"
 
+#ifndef JB_PC
+#define JB_PC 5
+#define JB_SP 4
+#define JB_BP 3
+#endif
+
 /* These are set once at boot time and not changed thereafter */
 
 static unsigned long exec_regs[HOST_FRAME_SIZE];
